src/senselab/audio/data_structures/audio.py

```diff
@@ -68,6 +74,15 @@ def from_filepath(cls, filepath: str | os.PathLike, metadata: Dict = {}) -> "Aud
             filepath: Filepath of the audio file to read from
             metadata: Additional information associated with the audio file
         """
+        if not TORCHAUDIO_AVAILABLE:
+            raise ImportError(
```

Comment:
Fairly pedantic but we should raise a `ModuleNotFoundError` since that is the more descriptive error in python 3.6+.